### PR TITLE
chore: update rs locales

### PR DESCRIPTION
**Pull request by radulle**

### CODE REVIEW

File: config/locales/rs.yml [hunk 1]
```diff
@@ -515,29 +515,29 @@ rs:
...
-          <p>In short, there is needed your desire.</p>
+          <p>Ukratko, potrebna je samo volja.</p>
           <p>
-            In more detail, you need:
+            Detaljnije, potrebni su:
             <ul>
-              <li>safe and comfortable route</li>
-              <li>a team that ready to handle runs every saturday all year round</li>
-              <li>a desire to make everyone healthier and happier</li>
+              <li>bezbedna i stazasafe and comfortable route</li>
```
Comment (vol1ura) [inline config/locales/rs.yml:527]: Maybe it would be better to completely translate this phrase to serbian without mix?

Comment (nradulaski-godaddy) [reply config/locales/rs.yml:527]: Yes, missed that, I guess comfortable meant that it easy to get to / access so I'll translate it that way.

Comment (radulle) [reply config/locales/rs.yml]: Yes, missed that, I guess comfortable meant that it easy to get to / access so I'll translate it that way.

**Comment by vol1ura** [comment]
There is a problem with encoding - application and tests are falling with exception 

```
I18n::InvalidLocaleData (can not load translations from /usr/src/config/locales/rs.yml: 
#<Psych::SyntaxError: (<unknown>): found character that cannot start any token while scanning for the next token at line 540 column 91>)
```
I think files should be in utf-8

File: config/locales/rs.yml [hunk 2]
```diff
@@ -515,29 +515,29 @@ rs:
...
-        title: S95 races in other countries
-        body: Today our run events are held also in %{countries}. All results are in one system saved and you can see it in athlete profiles.
+        title: S95 trke u drugim zemljama
+        body: Danas se naše trke organizuju i u %{countries}. Svi rezultati se čuvaju u jednom sistemu odakle ćete imati uvid u profile atletiča.
       back: Nazad
       forward: Napred
     rules:
       description: Brief rules for S95 participants, as well as frequently asked questions
-      register_with_bot_html: 'If you are not registered, create a profile using <a href="%{link}" target="_blank" rel="noopener">Telegram bot</a>. You will receive a link to your profile with a personal qr-code.'
+      register_with_bot_html: 'If you are not registered, create a profile using <a href='%{link}' target='_blank' rel='noopener'>Telegram bot</a>. You will receive a link to your profile with a personal qr-code.'
```
Comment (vol1ura) [inline config/locales/rs.yml:540]: Please, leave here double quotes, there is a parsing error

File: config/locales/rs.yml [hunk 3]
```diff
@@ -547,7 +547,7 @@ rs:
         answer: The track is well marked with orange cones, there is also a marshal at the turn (if there is one), and most likely you will be surrounded by other participants when running.
       about_registering:
         question: Do I need to register to participate in the run?
-        answer_html: 'It is not necessary, but it is advisable to go through the registration procedure using our telegram bot <a href="%{link}" target="_blank" rel="noopener">%{bot_nick}</a>. As a result, you will have a personal barcode, which you will subsequently show to the volunteer. In addition, the bot will send notifications with the results of the run.'
+        answer_html: 'It is not necessary, but it is advisable to go through the registration procedure using our telegram bot <a href='%{link}' target='_blank' rel='noopener'>%{bot_nick}</a>. As a result, you will have a personal barcode, which you will subsequently show to the volunteer. In addition, the bot will send notifications with the results of the run.'
```
Comment (vol1ura) [inline config/locales/rs.yml:550]: Also double quotes

File: config/locales/rs.yml [hunk 4]
```diff
@@ -660,13 +660,13 @@ rs:
   clubs:
     index:
       description: Table of clubs with statistics on the total number of participants, volunteers and runs
-      how_to_add_new_club_html: 'Is your club not on the list? Write us <a href="mailto:%{mail}?subject=Add a new club" target="_blank" title="Send us an email">email</a> and we will add it.'
-      name: Name
-      athletes: athletes
-      results: results
-      volunteering: volunteering
+      how_to_add_new_club_html: 'Is your club not on the list? Write us <a href='mailto:%{mail}?subject=Add a new club' target='_blank' title='Send us an email'>email</a> and we will add it.'
```
Comment (vol1ura) [inline config/locales/rs.yml:663]: Also double quotes

File: config/locales/rs.yml [hunk 5]
```diff
@@ -679,25 +679,25 @@ rs:
       attention: Pažnja!
       opening_soon: Uskoro otvaranje!
       start_cancelled: Sledeći start je otkazan.
-      inactivity_disclaimer_html: 'Pratite vesti u <a href="%{link}" class="alert-link">Telegram kanalu</a>, a u međuvremenu, dođite na ostale <a href="/" class=" alert-link">naše trke</a>.'
-      about_volunteers_html: 'Svi događaji su besplatni i organizovani od strane volontera. <a href="%{roster_link}">Možete pogledati dostupne pozicije i takođe se pridružiti organizacionom timu</a>.'
-      about_participation_html: '<a href="%{registration_link}" target="_blank" rel="noopener">Registrujte se</a> u našem botu i dođite u subotu ujutru. Distanca se može trčati ili hodati. Dobrodošli su svi, bez obzira na uzrast i nivo treninga.'
+      inactivity_disclaimer_html: 'Pratite vesti u <a href='%{link}' class='alert-link'>Telegram kanalu</a>, a u međuvremenu, dođite na ostale <a href='/' class=' alert-link'>naše trke</a>.'
```
Comment (vol1ura) [inline config/locales/rs.yml:682]: Here and below double quotes are needed

**Review by vol1ura — CHANGES_REQUESTED** [review]

File: config/locales/rs.yml (hunk 2 above)
Comment (radulle) [reply config/locales/rs.yml:540]: It seems that VSCode's auto-format kicked in without me noticing :(

**Comment by vol1ura** [comment]
@radulle Thank you very much for request and help!

**Comment by nradulaski-godaddy** [comment]
> @radulle Thank you very much for request and help!

No peoblem, I'll try to translate the rest, I might have some question about context of some, I'll send those to Discussions when ready.

**Comment by radulle** [comment]
> @radulle Thank you very much for request and help!

No problem, I'll try to translate the rest, I might have some question about context of some, I'll send those to Discussions when ready.

**Pull request merged**